rs/boundary_node/rate_limits/canister_client/src/main.rs

```diff
@@ -8,12 +8,16 @@ use rate_limits_api::{
     GetRuleByIdResponse, IncidentId, InputConfig, InputRule, RuleId, Version,
 };
 
-const RATE_LIMIT_CANISTER_ID: &str = "un4fu-tqaaa-aaaab-qadjq-cai";
+const RATE_LIMIT_CANISTER_ID: &str = "zkfwe-6yaaa-aaaab-qacca-cai";
 const IC_DOMAIN: &str = "https://ic0.app";
 
 use k256::elliptic_curve::SecretKey;
 
-const TEST_PRIVATE_KEY: &str = "";
+const TEST_PRIVATE_KEY: &str = "-----BEGIN EC PRIVATE KEY-----
```

Comment:
You could also use: https://github.com/dfinity/ic/blob/34b7182bf427278f065a1a194e62b1c023d2ae00/rs/test_utilities/identity/src/lib.rs#L30-L55